src/pages/dmt5/_download-app.js

```diff
@@ -20,6 +21,12 @@ const query = graphql`
         }
     }
 `
+
+const DownloadLinkWrapper = styled.div`
+    display: grid;
+    grid-template-columns : repeat(2, 1fr);
+    grid-gap: 0.8rem;
```

Comment:
should we use `px` here

Reply:
Yup man, now coming to think of it I think px would be a better choice 😂